resources/icons/light/diff.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 3.5L2.5 3H7.5L8 3.5V12.5L7.5 13H2.5L2 12.5V3.5ZM3 12H7V6H3V12ZM3 5H7V4H3V5ZM9.5 3H14.5L15 3.5V12.5L14.5 13H9.5L9 12.5V3.5L9.5 3ZM10 12H14V10H10V12ZM10 8H14V4H10V8Z" fill="#C5C5C5"/>
```

Comment:
These are a bit hard to see on a light theme - would be good to match the revert icon - I think this is right:

![image](https://user-images.githubusercontent.com/49367953/76348071-81145f80-62ff-11ea-9ed7-cc8758bf5072.png)

```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M2 3.5L2.5 3H7.5L8 3.5V12.5L7.5 13H2.5L2 12.5V3.5ZM3 12H7V6H3V12ZM3 5H7V4H3V5ZM9.5 3H14.5L15 3.5V12.5L14.5 13H9.5L9 12.5V3.5L9.5 3ZM10 12H14V10H10V12ZM10 8H14V4H10V8Z" fill="#424242"/>
```